Source/BluetoothManager.swift

```diff
@@ -265,7 +265,7 @@ public class BluetoothManager {
             strongSelf.centralManager.cancelPeripheralConnection(peripheral.peripheral)
             return disposable
         }
-        return ensure(.poweredOn, observable: observable).asSingle()
```

Comment:
Is there a reason for dropping the `Single` here? Can we emit multiple events now?

Reply:
Good catch! Already repairing it.